releases/release-1.16/README.md

```diff
@@ -0,0 +1,197 @@
+# Kubernetes 1.16
+
+#### Links 
+* [This document](https://git.k8s.io/sig-release/releases/release-1.16/README.md)
+* [Release Team](https://git.k8s.io/sig-release/releases/release-1.16/release_team.md)
+* [Meeting Minutes](http://bit.ly/k8s116-releasemtg) - TBD
+* [v1.16 Release Calendar](http://bit.ly/k8s116-cal)
+* Contact: [#sig-release] on slack, [kubernetes-release-team@] on e-mail
+
+#### Tracking docs
+
+* [Enhancements Tracking Sheet](http://bit.ly/k8s116-enhancements)
+* Bug Triage Tracking Sheet
+* [CI Signal Report](http://bit.ly/k8s116-cisignal)
+* [Retrospective Document](http://bit.ly/k8s116-retro)
+* [kubernetes/kubernetes v1.16 milestone](https://github.com/kubernetes/kubernetes/milestone/45)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## tl;dr
+The 1.16 release cycle is proposed as follows: 
+
+- **Monday, July 01**  - Release Cycle Begins 
+- **Tuesday, July 30, EOD PST** - Enhancements Freeze 
+- **Thursday, August 29, EOD PST**  - Code Freeze 
+- **Monday, September 9** - Docs must be completed and reviewed 
+- **Monday, September 16** - Kubernetes 1.16.0 Released 
+
+## What will we do differently this release? 
+* We have dissolved the test-infra role from the release team. The responsibilities of this role have been reallocted amongst other roles on the release team. This is the tracking [issue](https://github.com/kubernetes/sig-release/issues/631).
+* We have a new role on the release team - the emeritus lead.  
+
```

Comment:
I'd suggest highlighting that we're doing away with the changelog in the Markdown and exclusively using the relnotes site (Markdown will still have themes, deprecations, etc etc.)